src/openads/domain/position/PositionAdError.js

```diff
@@ -2,9 +2,11 @@ export default class PositionAdError extends Error {
   constructor({position = {}} = {}) {
     super()
     this.name = 'PositionAdError'
-    this.message = `Error loading Ad ${position.id}: ${(position.ad &&
-      position.ad.data &&
-      position.ad.data.errMessage) ||
+    let msg = position.ad && position.ad.data && position.ad.data.errMessage
+    if (!msg && typeof position.ad.data === 'string') {
```

Comment:
maybe this is not needed as the error already has the `position` attribute

Reply:
It's easy to read if you have in the first line. Unexpected error doesn't add any info.